backend/jottacloud/jottacloud.go

```diff
@@ -1406,7 +1406,46 @@ func (o *Object) ModTime(ctx context.Context) time.Time {
 
 // SetModTime sets the modification time of the local fs object
 func (o *Object) SetModTime(ctx context.Context, modTime time.Time) error {
-	return fs.ErrorCantSetModTime
+	// make sure metadata is available, we need its current size and md5
+	err := o.readMetaData(ctx, false)
+	if err != nil {
+		fs.Logf(o, "Failed to read metadata: %v", err)
+		return err
+	}
+
+	// prepare allocate request with existing metadata but changed timestamps
+	o.fs.tokenRenewer.Start()
+	defer o.fs.tokenRenewer.Stop()
+	var resp *http.Response
+	var options []fs.OpenOption
+	opts := rest.Opts{
+		Method:       "POST",
+		Path:         "files/v1/allocate",
+		Options:      options,
+		ExtraHeaders: make(map[string]string),
+	}
+	fileDate := api.Time(modTime).APIString()
+	var request = api.AllocateFileRequest{
+		Bytes:    o.size,
+		Created:  fileDate,
+		Modified: fileDate,
+		Md5:      o.md5,
+		Path:     path.Join(o.fs.opt.Mountpoint, o.fs.opt.Enc.FromStandardPath(path.Join(o.fs.root, o.remote))),
+	}
+
+	// send it
+	var response api.AllocateFileResponse
+	err = o.fs.pacer.CallNoRetry(func() (bool, error) {
```

Comment:
Why is this `NoRetry`? We generally use that for uploads which read from an `io.Reader` which can't be rewound.

Reply:
No idea! I copied it from the existing upload implementation without thinking about this. I guess I should just do `Call`. Will fix.

Reply:
Done. Also removed the `response api.AllocateFileResponse` object, as it was not checked anyway, sending nil as argument saves a json parsing I guess.

Reply:
I think I will re-add the response, and instead actually check it. I managed to provoke an error that went unnoticed, although not very likely in real life cases, its probably better safe than sorry.